src/components/TopBar/Notifications/Notifications.tsx

```diff
@@ -62,30 +64,34 @@ export const Notifications = ({ alertsQuantity }: NotificationsProps): JSX.Eleme
           <StyledIconWrapper>
             <Svg id="alert" width="16" height="20" />
           </StyledIconWrapper>
-          <StyledLabelLarge color={variables.palette.on_surface_variant} fontWeight="semiBold">
-            {t('alerts')}
-          </StyledLabelLarge>
+          <StyledLabelLarge fontWeight="semiBold">{t('alerts')}</StyledLabelLarge>
         </StyledHeaderLeft>
         <StyledFlexTopCenter>
-          <StyledLabelLarge color={variables.palette.on_surface_variant} fontWeight="semiBold">
-            {`${alertsQuantity} `}
-            {t('unread')}
+          <StyledLabelLarge fontWeight="semiBold" color={variables.palette.semantic.error}>
+            {`${alertsQuantity || ''} `}
```

Comment:
do we need } `} here?

Reply:
we need additional space on the end before 'unread'

Reply:
add before unread  { ' '} and before { alertsQuantity || ''}